tests/test.ts

```diff
@@ -8,6 +8,15 @@ QUnit.todo('testing todo', (assert) => {
 
 QUnit.skip('testing skip')
 
+QUnit.test('basic async test example', assert => {
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      assert.strictEqual(true, true, 'this is fine')
+      resolve()
+    }, 100)
+  })
+})
+
 QUnit.module('Group A', hooks => {
   // It is valid to call the same hook methods more than once.
   hooks.beforeEach(assert => {
```

Comment:
If i change an assertion to fail, the beforeEach and afterEach calls of the corresponding module are marked as failed as well in the summary. Is there some information contained in the data qunit reports to avoid this?

Reply:
This is bug on my end. I'll push up a fix shortly.